sample-apps/react-native/dogfood/src/screens/LoginScreen.tsx

```diff
@@ -28,24 +28,22 @@ GoogleSignin.configure({
   // profileImageSize: 120, // [iOS] The desired height (and width) of the profile image. Defaults to 120px
 });
 
-const generateValidUserName = (username: string) => {
-  return username
-    .replace(/[^_\-0-9a-zA-Z@]/g, '_')
-    .replace('@getstream_io', '');
+const generateValidUserId = (userId: string) => {
+  return userId.replace(/[^_\-0-9a-zA-Z@]/g, '_').replace('@getstream_io', '');
 };
 
 const LoginScreen = () => {
-  const [localUserName, setLocalUserName] = useState('');
+  const [localUserId, setLocalUserId] = useState('');
   const [loader, setLoader] = useState(false);
 
   const setState = useAppGlobalStoreSetState();
 
   const loginHandler = async () => {
     try {
-      const _username = generateValidUserName(localUserName);
-      const _userImageUrl = `https://getstream.io/random_png/?id=${_username}&name=${_username}`;
+      const _userId = generateValidUserId(localUserId);
+      const _userImageUrl = `https://getstream.io/random_png/?id=${_userId}&name=${_userId}`;
       setState({
-        username: _username,
+        userId: _userId,
         userImageUrl: _userImageUrl,
```

Comment:
We should set the userName as the userId here

else if userName is used to show up in participant view, it will break

Reply:
No, participant view has a condition of name || id